scapy/contrib/cansocket_python_can.py

```diff
@@ -95,29 +55,34 @@ def mux(self):
         object. If a message is received, this message gets forwarded to
         all receive queues of the SocketWrapper objects.
         """
+        msgs = []
```

Comment:
Deque here might also allow for stuff like efficient ReceiveSelector (last, first or all messages at once) to happen or don't I understand eventual change?

Reply:
I don't see much advantage in using deque in this position. We're coping all messages in one list first, then we copy this list to all Sockets.

Reply:
Okay, thanks for explanation, LGTM